app/denom_units_test.go

```diff
@@ -0,0 +1,108 @@
+//go:build !experimental
+// +build !experimental
+
+package app
+
+import (
+	"encoding/json"
+	"os"
+	"testing"
+
+	"github.com/CosmWasm/wasmd/app"
```

Comment:
It looks like `app` is being used for `DefaultNodeHome` and `DefaultConsensusParams`. `DefaultNodeHome` should be available within this package (i.e. without using the `wasmd` app package) and `DefaultConsensusParams` does not need to be set for this unit test nor do I think we should be using the `wasmd` package for this.